toDo.txt

```diff
@@ -0,0 +1,3 @@
+Review possible additional test cases for articles/:article_id
+    Add description to endpoints.json
+    When completed do a pull request for this feature and delete branch
```

Comment:
This wouldn't want to be part of your PR since it's just a 'note to self', I'd gitignore it